docs/guides/kubernetes-dashboard.md

```diff
@@ -14,359 +14,100 @@ The following guide covers how to secure [Kubernetes Dashboard] using Pomerium.
 
```

Comment:
[copy] intro blurb is no longer accurate.  We don't cover forward auth.  It's also worth calling out that we're able to pass the user's identity to the API server through the dashboard.  This is in contrast to most deployments that use static tokens.

Reply:
Adjusted in https://github.com/pomerium/pomerium/pull/2759/commits/55f216691b1af0726fa74b3b0ad1bfd4c8f81257